sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-assets/test

```diff
@@ -1,12 +1,12 @@
 # the server should properly start
 > fastOptJS::webpack
-# 3 assets and 2 for loader and entry
-> html index.html 5
+# 1 assets and 3 for library, loader and app
```

Comment:
> Stats differentiate between files and auxiliaryFiles now
[Source](https://webpack.js.org/blog/2020-10-10-webpack-5-release/))

i.e. map files are now included in returned assets anymore (breaking change ⚠️). Same for associated production test below